### DISCUSSION TITLE
"make" does not work

### DISCUSSION THREAD

**Discussion by HiccopABC** [tendermint/tendermint#10083] [General]
![snipaste_20231130_222928](https://github.com/tendermint/tendermint/assets/107093768/c9c53a77-e91d-4954-a5ac-572bd8349ac5)
When I run "make build" or other "make" commands , I get a makefile build error,How do I solve this?

**Comment by sergio-mena** [comment]
@HiccopABC you can try https://github.com/cometbft/cometbft/, which is a fork of Tendermint
